kv/etcd/store.go

```diff
@@ -522,8 +522,13 @@ func newValue(val []byte, ver, rev int64) *value {
 	}
 }
 
-func (c *value) isNewer(other *value) bool {
-	return c.Rev > other.Rev
+func (c *value) IsNewer(other kv.Value) bool {
+	othervalue, ok := other.(*value)
```

Comment:
yeah for example if a user decides to compare this *value and *mem.value (although no one really should do that) this check would come in handy.